packages/analyticsdx-template-lint/src/schemas/template-info-schema.json

```
@@ -679,6 +679,30 @@
                 "body": "${0}"
               }
             ]
+          },
+          "liveConnection": {
+            "type": "object",
+            "additionalProperties": false,
```

Turn off auto-suggest in the editor for now:
```suggestion
            "additionalProperties": false,
            "$comment": "For now, allow this to be specified, but do not offer it in code-completions until the feature is ready",
            "doNotSuggest": true
```